.github/workflows/build-keystone-operator.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17.x
+        go-version: 1.18.0
```

Comment:
As I mentioned in another PR from Andrew, I think in a follow up we should get this version from go.mod to not have to care about when things change there.